appengine/flexible/websockets/templates/index.html

```diff
@@ -0,0 +1,98 @@
+{#
+# Copyright 2015 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#}
+{# [START index] #}
+<!doctype html>
+<html>
+  <head>
+    <title>Google App Engine Flexible Environment - Python Websockets Chat</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  </head>
+  <body>
+
+    <!-- [START form] -->
```

Comment:
This is an awfully generic region tag, which makes it hard to track internally. :)

Is it possible to use something more self-explanatory? (e.g. `gae_websockets_form`?)

Reply:
Good call, fixed